pages/_app.tsx

```diff
@@ -23,7 +22,7 @@ function MyApp({ Component, pageProps }: AppProps): JSX.Element {
   const start = () => NProgress.start();
   const end = (url) => {
     NProgress.done();
-    if (isProduction) gtag.pageview(url);
+    if (gtag.isProduction) gtag.pageview(url);
```

Comment:
Do we need this if statement here if you're already checking for isProduction in the pageview function?

Reply:
Removed the if statement!